Connections/Shear/Endplate/endPlateCalc.py

```diff
@@ -171,31 +181,36 @@ def end_connection(ui_obj):
     beam_fy = float(ui_obj['Member']['fy (MPa)'])
               
     shear_load = float(str(ui_obj['Load']['ShearForce (kN)']))
-    print " shear_load =", shear_load
-                  
+
     bolt_dia = int(ui_obj['Bolt']['Diameter (mm)'])
     bolt_type = ui_obj["Bolt"]["Type"]
     bolt_grade = float(ui_obj['Bolt']['Grade'])
               
     end_plate_t = float(ui_obj['Plate']['Thickness (mm)'])
-    end_plate_w = ui_obj['Plate']['Width (mm)']
+    end_plate_w = str(ui_obj['Plate']['Width (mm)'])
     if end_plate_w == '':
         end_plate_w = 0
-    end_plate_l = ui_obj['Plate']['Height (mm)']
+        
+    end_plate_l = str(ui_obj['Plate']['Height (mm)'])
     if end_plate_l == '':
         end_plate_l = 0
-    web_plate_fu = ui_obj['Member']['fu (MPa)']
-    web_plate_fy = ui_obj['Member']['fy (MPa)']
+        
+    web_plate_fu = float(ui_obj['Member']['fu (MPa)'])
```

Comment:
If these conversion hasn't been implemented upstream, then these should be:
 float(str(****)) 
As the string literals are stored as unicode.